transmart-core-api/src/main/groovy/org/transmartproject/core/multidimquery/MultiDimensionalDataResource.groovy

```diff
@@ -31,17 +27,23 @@ interface MultiDimensionalDataResource {
     Dimension getDimension(String name)
 
     Long count(MultiDimConstraint constraint, User user)
+
     Long cachedCount(MultiDimConstraint constraint, User user)
 
     Iterable getDimensionElements(Dimension dimension, MultiDimConstraint constraint, User user)
 
-    QueryResult createPatientSet(String name, MultiDimConstraint constraint, User user, String constraintText, String apiVersion) 
+    QueryResult createPatientSetQueryResult(String name, MultiDimConstraint constraint, User user, String constraintText, String apiVersion)
+
+    QueryResult createObservationSetQueryResult(String name, User user, String constraintText, String apiVersion)
```

Comment:
constraint as text but not as typed object?

Reply:
@gijskant We use it to store the constraint in `qtQueryMaster.requestConstraints` field.